lbz/request.py

```diff
@@ -68,6 +50,43 @@ def _safe_json_loads(payload: str | bytes) -> Any:
         except ValueError as error:
             raise BadRequestError(f"Invalid payload.\nPayload body:\n {repr(payload)}") from error
 
+
+class Request(BaseRequest):
```

Comment:
If you already changed the version to 0.7.0, you can rename this class instead of creating the alias below 😉